app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/codePager/SampleInfoScreen.kt

```diff
@@ -179,7 +182,10 @@ private fun CodePageView(
                         .height(200.dp)
                         .width(350.dp)
                         .padding(8.dp),
-                    model = screenshotURL,
+                    model = ImageRequest.Builder(LocalContext.current)
+                        .data(screenshotURL)
+                        .crossfade(true)
+                        .build(),
```

Comment:
Added a crossfade-in to the Coil aysc image view to allow the WebView initialize to render with less UI load.

Reply:
Looks good and no crash on my API 36 device 👍